atom/browser/browser.h

```diff
@@ -156,7 +156,7 @@ class Browser : public WindowListObserver {
   };
 
   // Add a custom task to jump list.
-  void SetUserTasks(const std::vector<UserTask>& tasks);
+  bool SetUserTasks(const std::vector<UserTask>& tasks);
```

Comment:
I have also noticed that both app.setAsDefaultProtocolClient(protocol) and app.removeAsDefaultProtocolClient(protocol) return a boolean, I've updated the docs.